Turf/Turf.swift

```diff
@@ -285,3 +288,67 @@ public struct Polyline {
         return closestCoordinate
     }
 }
+
+public struct Polygon {
+    var polygonCoordinates: [[CLLocationCoordinate2D]]
```

Comment:
GeoJSON represents a polygon as a collection of rings (one outer ring and multiple inner rings). Similarly, the Mapbox Maps SDKs for iOS and macOS represent a polygon as an outer polygon containing multiple inner polygons. Either way, it’s nice to be able to work with a formal structure instead of a nested array. Consider replacing this property with:

```swift
var rings: [Ring]
```

or:

```swift
var outerRing: Ring
var innerRings: [Ring]
```

where `Ring` is either a struct or a type alias for `[CLLocationCoordinate2D]`. As a struct, it would be a good home for the `ringArea(_:)` method.